### ISSUE TITLE
Ambiguity on the method used

### ISSUE THREAD

**Issue by Tangolin** [POSTECH-CVLab/PyTorch-StudioGAN#204]
The link to conditional batch optimization leads me to the paper "A learned Representation for Artistic Styles", however the technique proposed in that paper is conditional instance normalization, conditional batch normalization seems to be from "Modulating early visual processing by language", can I clarify which technique is the actual one used by the repository?

**Comment by mingukkang** [comment]
sorry for the confusion.

Please refer to [Modulating early visual processing by language](https://paperswithcode.com/paper/modulating-early-visual-processing-by) instead of A learned Representation for Artistic Styles.

**Comment by Tangolin** [comment]
Got it, thank you!
